### ISSUE TITLE
Add setting to disable path completion

### ISSUE THREAD

**Issue by lardissone** [withfig/fig#588]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my feature

### Description:

I like Fig, but I'm used to zsh path completion which I think is faster and works for me, so it would be great to be able to disable just this feature on Fig.

**Comment by brendanfalk** [comment]
Yes think we can add a setting for this. In the meantime, you can disable an entire CLI tool. 

Run this then restart.
```bash
fig settings autocomplete.disableForCommands '["npm","ls","cd"]'
```

I know this isn't the same, but it's a start
